webapp/src/editortoolbar.tsx

```diff
@@ -425,18 +430,12 @@ export class EditorToolbar extends data.Component<ISettingsProps, EditorToolbarS
         }
 
         return <div id="editortools" className="ui" role="region" aria-label={lf("Editor toolbar")}>
-            <div id="downloadArea" role="menubar" className="ui column items">{headless &&
-                <div className="ui item">
-                    <div className="ui icon large buttons">
-                        {compileBtn && <EditorToolbarButton icon={downloadIcon} className={`primary large download-button mobile tablet hide ${downloadButtonClasses}`} title={compileTooltip} onButtonClick={this.compile} view='computer' />}
-                    </div>
-                </div>}
-                {/* TODO clean this; make it just getCompileButton, and set the buttons fontsize to 0 / the icon itself back to normal to just hide text */}
-                {!headless && !isTimeMachineEmbed && <div className="ui item portrait hide">
```

Comment:
why are you removing the check for headless and time machine?

Reply:
Possibly bad merge. Will restore these flags.

Reply:
Restored `isTimeMachineEmbed`. I couldn't simply re-add `headless` because bedrock is headless yet does want the download button to display. I think I was able to adjust the logic so it works for all targets. Tested in Arcade, Microbit, and Minecraft. All behaved like they do in production.

Reply:
@eanders-ms did you have all the URL flags for minecraft though? i think the download button is hidden in minecraft...

Reply:
Here's what it looks like in EDU: 
![image](https://github.com/user-attachments/assets/0f2755b7-7f7e-417b-8bce-b584273a581a)